bokeh/properties.py

```diff
@@ -175,7 +175,8 @@ def matches(self, new, old):
         except (KeyboardInterrupt, SystemExit):
             raise
         except Exception as e:
-            logger.debug("could not compare %s and %s for property %s (Reason: %s)", new, old, self.name, e)
+            # if we cannot compare (e.g. arrays) just punt return False for match
+            pass
```

Comment:
I was thinking about broken implementations of `__eq__` or something like that. It wouldn't be a common thing, for sure.